released/packages/coq-commutative-diagrams/coq-commutative-diagrams.0.1/opam

```diff
@@ -0,0 +1,35 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "A Coq plugin to deal with commutative diagrams"
+maintainer: ["luc@dwarfmaster.net"]
+authors: ["Luc Chabassier"]
+license: "MIT"
+homepage: "https://github.com/dwarfmaster/commutative-diagrams"
+bug-reports: "https://github.com/dwarfmaster/commutative-diagrams/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.08"}
+  "coq-core" {>= "8.15" & < "8.16~"}
```

Comment:
I just fixed the `coq-core` one, not sure why I haven't run into it until now. I should depend on `coq-hott`, but for some reason `opam install .` succeeded without it and I haven't though about that since. I'll add it.